src/solvers/refinement/README.md

```diff
@@ -6,6 +6,22 @@
 
 \section string_solver_interface String solver interface
 
+After the input program of CProver has been converted to a goto-program and then
+to a set of equations, these equations are fed to a solver.
+For program verification, this is generally the last step in CProver, but we can
+for instance ask for a trace to be generated from the result of the solver.
```

Comment:
For model checking, not generating a trace isn't really an option?

Reply:
cbmc and jbmc do not generate a trace if you don't give the `--trace` option.

Reply:
Agreed, but it was not at all clear to me at what level this text is being pitched. The very general term "program verification" is being used, which seems way less specific than talking about two tools (cbmc and jbmc).